java-frontend/src/test/java/org/sonar/java/model/JUtilsTest.java

```diff
@@ -235,18 +235,18 @@ class IsAnnotation {
 
     @Test
     void annotation() {
-      assertThat(JUtils.isAnnotation(anno.symbol())).isTrue();
+      assertThat(anno.symbol().isAnnotation()).isTrue();
```

Comment:
I think, technically this test (and the tests below) should now be moved to SymbolsTest, because we are now not testing JUtils anymore, but Symbol functions. Wdyt?